src/contentFetcher.ts

```diff
@@ -102,9 +102,9 @@ export class ContentFetcher {
         }
 
         const {baseEndpointUrl} = configuration;
-        const apiKey = configuration.apiKey !== undefined
-            ? ApiKey.from(configuration.apiKey)
-            : undefined;
+        const apiKey = typeof configuration.apiKey === 'object'
+            ? configuration.apiKey.getIdentifier()
+            : configuration.apiKey;
```

Comment:
I'd invert this, check if `typeof` is `'string'`.

Reply:
I thought about this, but I'd have to handle the undefined case in the else branch.